### PR TITLE
Sidebar and login page fix

### PR DESCRIPTION
**Pull request by dipujscoder**
Fixed issue #43 #42 #40 #26 #24

### CODE REVIEW

File: src/signup1.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@@include('header.htm', { "title": "Sleek - Admin Dashboard Template", "className": "header-fixed sidebar-fixed sidebar-dark header-light" })
```
Comment (iamabdus) [inline src/signup1.html]: Can you please explain why this file exist?

File: src/_inc/sidebar-with-footer.htm [hunk 2]
```diff
@@ -2,13 +2,18 @@
     ——— SIDEBAR WITH FOOTER
     ===================================== -->
     <div class="layout-sidenav bg-sidebar ">
-      <div class="mobile-sticky-body-overlay"></div>
+      <!-- <div class="mobile-sticky-body-overlay"></div> -->
 
       <div id="sidebar" class="sidebar sidebar-with-footer">
         <!-- begin sidebar scrollbar -->
         <div class="app-brand">
           <a class="logo" href="index.html">
```
Comment (iamabdus) [inline src/_inc/sidebar-with-footer.htm]: Logo is not properly centered

File: src/index.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@@include('header.htm', {	"title": "Sleek - Admin Dashboard Template",	"className": "header-fixed sidebar-fixed sidebar-dark header-light" })
+@@include('header.htm', {	"title": "Sleek - Admin Dashboard Template",	"className": "header-fixed sidebar-static-offcanvas sidebar-dark header-light" })
```
Comment (iamabdus) [inline src/index.html]: Why offcanvas?

File: src/assets/scss/_grid.scss [hunk 4]
```diff
@@ -90,3 +82,11 @@
     -ms-flex-pack: justify;
     justify-content: space-between;
 }
+
+/* Authentication Wrapper */ 
+.authentication {
```
Comment (iamabdus) [inline src/assets/scss/_grid.scss]: Why you named it .authentication?

File: src/signup1.html (hunk 1 above)
Comment (dipujscoder) [reply src/signup1.html]: That was unnecessary file. I have deeded that file

File: src/_inc/sidebar-with-footer.htm (hunk 2 above)
Comment (dipujscoder) [reply src/_inc/sidebar-with-footer.htm]: Ok i see this issue

File: src/index.html (hunk 3 above)
Comment (dipujscoder) [reply src/index.html]: Sorry! I was testing with offcanvas.

File: src/_inc/sidebar-with-footer.htm (hunk 2 above)
Comment (iamabdus) [reply src/_inc/sidebar-with-footer.htm]: fix the brand name display on minimized sidebar

**Comment by dipujscoder** [comment]
Please check #44

**Comment by dipujscoder** [comment]
Please check # issue

**Pull request merged**